Dockerfile

```diff
@@ -3,12 +3,10 @@ FROM ubuntu:latest
 RUN apt-get update && \
     apt-get upgrade -y && \
     apt-get install -y gcc-mingw-w64-i686 make git python3-pip && \
-    python3 -m pip install pyjson5
+    python3 -m pip install pyjson5 meson ninja
 
 RUN mkdir /app
 ADD . /app
 WORKDIR /app
 
-RUN chmod 0666 /app/src/init.c
-
-CMD ["make", "build"]
+CMD ["make", "debug_linux", "build"]
```

Comment:
This includes "make clean" which tries to remove the "build" directory from inside Docker, which it has no permissions for (if it even exists).

Reply:
"generate_init" lacks permissions to write to `src/init.c` under Docker (only used when the in-repo `init.c` is outdated)

Reply:
Fixed